charts/templates/statefulset.yaml

```diff
@@ -144,6 +144,31 @@ spec:
           {{- with .Values.resources }}
           resources: {{- toYaml . | nindent 12 }}
           {{- end }}
+        {{- if .Values.logCollection.enabled }}
+        - name: fluent-bit
+          image: fluent/fluent-bit:1.9.7
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: datadir
+              mountPath: {{ .Values.persistence.mountPath }}
+            {{- if .Values.extraVolumeMounts }}
+              {{- toYaml .Values.extraVolumeMounts | nindent 12 }}
+            {{- end }}
+            - name: {{ include "marklogic.fullname" . }}-fb-config-map
+              mountPath: /fluent-bit/etc/
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+          resources:
```

Comment:
The resource limit should also be configurable thru values file

Reply:
Resource limits change has been made and the other part of this comment is being tracked in another thread, marking as resolved